src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -28,8 +30,10 @@ public XunitPerformanceHarness(string[] args)
             _args = args;
             _disposed = false;
             _outputFiles = new List<string>();
+            _performanceTestConfig = new PerformanceTestConfig();
```

Comment:
I think this is just used for scenario tests.  We should name this ScenarioTestConfiguration so that it matches the RunScenario method.